src/code/templates/fv-egg-hatch.js

```diff
@@ -0,0 +1,59 @@
+import React, {PropTypes} from 'react';
+import FVAnimatedSprite from '../components/animated-sprite';
```

Comment:
There isn't really anything FableVision-specific about the `AnimatedSprite` component, and it's in the `components` folder rather than `fv-components`, so I'd be inclined to remove the `FV` and just call it the `AnimatedSprite` component.